Dockerfile

```diff
@@ -61,21 +61,14 @@ RUN yes | ./init/initial_ros_setup.sh --no-ros2 --download-only
 WORKDIR /ros2_ws/navigation2_ws/src/navigation2
 
 # change to correct branch if $BRANCH is not = master
-ARG BRANCH=master
-ARG REMOTE=origin
-RUN echo "Remote is $REMOTE"
-RUN if [ "$BRANCH" == "master" ]; \
+ARG PULLREQ=false
+RUN echo "pullreq is $PULLREQ"
+RUN if [ "$PULLREQ" == "false" ]; \
     then \
-      echo "On master branch"; \
+      echo "No pull request given"; \
```

Comment:
Does this mean it will build master or not build anything? Maybe it should say "No pull request given. Building master branch"?

Reply:
If it's a pull request, it will check it out before building it. If not, it builds master by default, since that is what happens when the repo is cloned in buildscript. I'll update the message though, to make it more clear.